jenkins-build.sh

```diff
@@ -259,6 +261,18 @@ if [ $cleanup_only -eq 1 ]; then
     set -e
 fi
 
+is_producer_running=1
+$USE_SUDO systemctl is-active s3backgroundproducer 2>&1 > /dev/null || is_producer_running=0
+if [[ $is_producer_running -eq 1 ]]; then
```

Comment:
if we intend to stop background services at this point ,can we simply stop them without this conditional check.